.github/workflows/verify.yaml

```diff
@@ -46,3 +46,9 @@ jobs:
         run: mvn test install -Dinvoker.skip -Dmaven.javadoc.skip=true -B -Dapiman.gateway-test.config=vertx3-file
       - name: Run Tests (AMG 1)
         run: mvn test install -Dinvoker.skip -Dmaven.javadoc.skip=true -B -Dapiman.gateway-test.config=amg-1
+      - name: Build and Publish Docker Images
+        if: github.ref == 'refs/heads/master'
+        run: |
+          mvn clean package docker:build -P docker -DskipTests
```

Comment:
Have you tried this locally?  Does it work?  I didn't think we had support for building a docker image using maven.  But looking at the code I *was* able to find a dockerfile and some configuration in the pom.xml.  So I guess this works?  If you can confirm that this produces what we want, then I can add the secrets and merge this PR.  It looks good.

Reply:
Yes, this should work after the project build has passed. But I am not sure if the `github.ref == 'refs/heads/master'` part is correct

Reply:
Yeah that should be fine - it just limits the step to the `master` branch.  Which I think is fine for us.